mgmt/tribe/worker/worker.go

```diff
@@ -332,23 +332,27 @@ func (w worker) loadPlugin(plugin core.Plugin) error {
 			}).Info("plugin not found")
 			continue
 		}
+
 		if resp.StatusCode == 200 {
 			if resp.Header.Get("Content-Type") != "application/x-gzip" {
 				logger.WithField("content-type", resp.Header.Get("Content-Type")).Error("Expected application/x-gzip")
 			}
 			dir, err := ioutil.TempDir("", "")
 			if err != nil {
 				logger.Error(err)
+				resp.Body.Close()
```

Comment:
So maybe move this part to separate function (and make this one less blobby), or just wrap it by ad hoc function
```
func() {
    ...
    resp := Do()
    iferr...
    defer resp.Body.Close()
    ...
}()
```